### PR TITLE
fix responsive image from team page. from issue #623

### PR DESCRIPTION
**Pull request by adiwahyudi02**
Fix responsive image from team page.
Solved #623

### CODE REVIEW

**Comment by adiwahyudi02** [comment]
result

**Review by pranav1698 — COMMENTED** [review]
Please write a description to your pr and link an issue which you have solved.

**Comment by adiwahyudi02** [comment]
Could you please check it now? @pranav1698 
https://adiwahyudi02.github.io/fossasia.org/team/

File: css/theme-lava.css
Comment (pranav1698) [inline css/theme-lava.css]: Same here

File: css/theme-lava.css
Comment (pranav1698) [inline css/theme-lava.css]: No need to add another block of code 
You can insert your changes in the same block

**Review by pranav1698 — CHANGES_REQUESTED** [review]

**Comment by adiwahyudi02** [comment]
I have updated it. please review again @pranav1698

**Review by pranav1698 — APPROVED** [review]

**Comment by pranav1698** [comment]
Please wait for someone with write access to merge the pr
@mariobehling

**Pull request merged**
